src/App/Documentation/components/Card/index.js

```diff
@@ -1,58 +1,106 @@
+            When the window reaches <Link to="/docs/core/breakpoints">breakpoint</Link> sm the cards will switch to mobile view and take up one row each and stretch to fill the width of the container.
+        </p>
         <ComponentPreview language="html" showCasePanel codeFigure>
             <div className="cards">
-                <ul>
-                    <CardComponent
-                        type="default"
-                        title="Card default"
-                        imgSrc="https://picsum.photos/300"
-                        icon="account_balance"
-                        highlightTitle="725 NOK"
-                        highlightSubTitle="Outstanding balance"
-                        text="This is the cards text"
-                        smallText="And the small text"
-                        btnTxt="Button"
-                        footerTxt="This is some text that goes in the footer."
-                        footerLinkTxt="You wanna click this link?"
-                    />
-                    <CardComponent
-                        type="brand"
-                        title="Card brand"
-                        imgSrc="https://picsum.photos/300"
-                        icon="face"
-                        highlightTitle="Bob Corlsan"
-                        highlightSubTitle="bob.corlsan@payex.com"
-                        text="This is the cards text"
-                        smallText="And the small text"
-                        btnTxt="Button"
-                        footerTxt="This is some text that goes in the footer."
-                        footerLinkTxt="You wanna click this link?"
-                    />
-                    <CardComponent
-                        type="light"
-                        title="Card light"
-                        imgSrc="https://picsum.photos/300"
-                        icon="verified_user"
-                        highlightTitle="75%"
-                        highlightSubTitle="Complete"
-                        text="This is the cards text"
-                        smallText="And the small text"
-                        btnTxt="Button"
-                        footerTxt="This is some text that goes in the footer."
-                        footerLinkTxt="You wanna click this link?"
-                    />
-                </ul>
+                <CardComponent
+                    type="default"
+                    title="Card default"
+                    imgSrc="https://picsum.photos/300"
+                    text="This is the cards text"
+                    smallText="And the small text"
+                    btnTxt="Button"
+                    footerTxt="This is some text that goes in the footer."
+                    footerLinkTxt="You wanna click this link?"
+                >
+                    <MediaObject size="sm" icon="account_circle" heading="Bob Corlsan" textsm="bob.corlsan@payex.com" />
```

Comment:
You have to change this to
```suggestion
                    <MediaObject size="sm" icon="account_circle" heading="Bob Corlsan" text="bob.corlsan@payex.com" textSmall />
```
to actually get small text now.